src/DssSpell.sol

```diff
@@ -40,154 +44,65 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE = ;
-    uint256 internal constant SIX_PT_FOUR_FIVE_PCT_RATE  = 1000000001982027061559507021;
-    uint256 internal constant SIX_PT_FOUR_NINE_PCT_RATE  = 1000000001993940198563273844;
-    uint256 internal constant SIX_PT_SEVEN_PCT_RATE      = 1000000002056410844314321266;
-    uint256 internal constant SIX_PT_SEVEN_FOUR_PCT_RATE = 1000000002068296073857195778;
-    uint256 internal constant SIX_PT_NINE_ONE_PCT_RATE   = 1000000002118758660201099744;
-    uint256 internal constant SEVEN_PT_ONE_SIX_PCT_RATE  = 1000000002192822766493423465;
-    uint256 internal constant SEVEN_PT_TWO_PCT_RATE      = 1000000002204656986467871801;
-    uint256 internal constant SEVEN_PT_TWO_FOUR_PCT_RATE = 1000000002216486791512316847;
 
     // ---------- Math ----------
-    uint256 internal constant THOUSAND = 10 ** 3;
-    uint256 internal constant RAY      = 10 ** 27;
-    uint256 internal constant RAD      = 10 ** 45;
-
-    // ---------- Reduce PSM-PAX-A Debt Ceiling & Disable DC-IAM ----------
-    VatAbstract internal immutable vat = VatAbstract(DssExecLib.vat());
-
-    // ---------- RETH-A Offboarding Parameters Finalization ----------
-    address internal immutable MCD_SPOT = DssExecLib.spotter();
-
-    // ---------- SBE parameter changes ----------
-    address internal immutable MCD_VOW  = DssExecLib.vow();
-    address internal immutable MCD_FLAP = DssExecLib.flap();
-
-    // ---------- Approve HV Bank (RWA009-A) DAO Resolution ----------
-    // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/21
-
-    // Approve DAO resolution hash QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w
-    // Note: by the previous convention it should be a comma-separated list of DAO resolutions IPFS hashes
-    string public constant dao_resolutions = "QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w";
-
-    address internal constant RWA009_A_INPUT_CONDUIT_URN_USDC = 0xddd021b7e3Bfbad19c7D455EB7976DCe51180141;
-    address internal immutable MCD_ESM                        = DssExecLib.esm();
+    uint256 internal constant MILLION = 10 ** 6;
+
+    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+    // Note: Function from https://github.com/makerdao/spells-goerli/blob/cd91b3e0ce234038d2e0ae047261177afac6f03c/archive/2024-01-12-DssSpell/DssSpell.sol#L54
```

Comment:
Nit: the comment suppose to link the original source of this function, not the last occurrence of it 🙂

Reply:
Updating it to the original source.